### ISSUE TITLE
Output files use asset role

### ISSUE THREAD

**Issue by Deraen** [adzerk-oss/boot-cljs#89]
Cljs compiler currently needs list of all directories to currently detect changes. This causes problems are the directories can contain cljs code (source maps) writen by other cljs tasks.

https://github.com/adzerk-oss/boot-cljs/commit/6329cd19e0a9bf82e69b7f426eaa020c080740b6
https://github.com/adzerk-oss/boot-cljs/commit/0c7fac5358a645e8ea471990e343d414f98cb3e9

It would be possible to mark files created by cljs task in fileset and then remove those files from directories before cljs task runs the compiler.

Example by @micha:
```clj
(let [fileset' (->> fileset (remove-when-meta [::cljs]) commit!)]
  ;; marked files don't exist in input-dirs now
  (do-cljs-compiling (input-dirs fileset'))
  ;; update the original fileset, all files are added back
  (->> fileset (add-compiled-with-meta tmpdir ::cljs) commit!))
```

We'll keep current behavior of using asset role until we know if Cljs compiler is working as indented. If needed, we can implement the workaround.

**Comment by Deraen** [comment]
Reverted the change so that it's possible to server CLJS output files from classpath.

**Comment by Deraen** [comment]
Implemented and merged #90.
